doc/dev/testing/running-the-tests.md

```diff
@@ -0,0 +1,129 @@
+# Running the SDK Tests
+
+The operator-sdk is automatically tested with a variety of tests anytime
```

Comment:
Nit: My personal preference is that we don't force a new line after x amount of characters for markdown.

Reply:
I don't think it should be forced, but I feel that it makes it easier to review if the lines are split up more frequently.